2021-11-12 00:04:37,334 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2021-11-12 00:04:37,335 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/instabot/bot/bot.py
2021-11-12 00:04:51,942 - instabot version: 0.117.0 (bot) - INFO - Bot stopped. Worked: 0:00:14.607957
2021-11-12 00:04:51,943 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2021-11-12 00:04:53,565 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-11-12 00:04:53,566 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-11-12 00:04:53,566 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-11-12 00:04:53,566 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
